1.30 breaks when using anyhow(?): `std::error::Error` is not implemented for `wasmtime::Error`

Just tried to update to 1.30 and ended up with lots of these errors:
```
error[E0277]: `?` couldn't convert the error: `wasmtime::Error: std::error::Error` is not satisfied
   --> C:\Users\Oscar\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\extism-1.30.0\src\current_plugin.rs:209:50
    |
209 |         if let Some(d) = xs.data_mut(&mut *store)? {
    |                             ---------------------^ the trait `std::error::Error` is not implemented for `wasmtime::Error`
    |                             |
    |                             this has type `Result<_, wasmtime::Error>`
    |
    = note: the question mark operation (`?`) implicitly performs a conversion on the error value using the `From` trait
    = note: required for `extism_convert::Error` to implement `From<wasmtime::Error>`

error[E0277]: `?` couldn't convert the error: `wasmtime::Error: std::error::Error` is not satisfied
   --> C:\Users\Oscar\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\extism-1.30.0\src\current_plugin.rs:238:14
    |
238 |               )?;
    |                ^ the trait `std::error::Error` is not implemented for `wasmtime::Error`
    |
   ::: C:\Users\Oscar\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\extism-1.30.0\src\lib.rs:5:33
    |
  5 |       ($store: expr, $x:expr) => {{
    |  _________________________________-
  6 | |         let y = $x;
  7 | |         if y.is_err() && $store.get_fuel().is_ok_and(|x| x == 0) {
  8 | |             Err(wasmtime::Error::msg("plugin ran out of fuel"))
...   |
 12 | |     }};
    | |_____- this has type `Result<_, wasmtime::Error>`
    |
    = note: the question mark operation (`?`) implicitly performs a conversion on the error value using the `From` trait
    = note: required for `extism_convert::Error` to implement `From<wasmtime::Error>`
```

My guess is that it relates to #901 @zoosky 

I depend on extism with `"default-features" = false`, but the tests pass with this so not really sure what happens.

I note that `anyhow` is no longer imported in `wasmtime` (from `Cargo-lock`), so I guess that maybe there should be an `anyhow` feature in `extism` that enables `wasmtime/anyhow` or add the `anyhow` feature in `wasmtime` by default.

Edit: it is possible to run with default features or add the feature `wasmtime-default-features`, but that also adds quite a bit of additional dependencies.

--- a/runtime/Cargo.toml
+++ b/runtime/Cargo.toml
@@ -10,7 +10,7 @@ version.workspace = true
 
 [dependencies]
 wasmtime = { version = "43", default-features = false, features = [
-  # 'anyhow',
+  'anyhow',
   'cache',
   'gc',
   'gc-drc',

--- a/runtime/Cargo.toml
+++ b/runtime/Cargo.toml
@@ -10,6 +10,7 @@ version.workspace = true
 
 [dependencies]
 wasmtime = { version = "43", default-features = false, features = [
+  # 'anyhow',
   'cache',
   'gc',
   'gc-drc',

